tests/local/Earthfile

```diff
@@ -90,6 +90,30 @@ test-save-local-artifact-with-workdir:
     COPY +save-local-artifact-with-workdir/data .
     RUN cat data | grep '^ddd$'
 
+#################################################################################################################
+# tests for testing overwriting existing local files
+
+some-artifact:
+    RUN printf '4321' > four-bytes.txt
+    SAVE ARTIFACT four-bytes.txt
+
+test-local-overwrites:
```

Comment:
can you add a `BUILD +test-local-overwrites` to the `all` target, which will then run this with our integration tests.

Reply:
whoops, and done